src/Microsoft.Framework.DesignTimeHost/ApplicationContext.cs

```diff
@@ -823,7 +877,7 @@ private void WriteAssembly(int version, ProjectWorld project, BinaryWriter write
                 var obj = new JObject();
                 obj["MessageType"] = "Assembly";
                 obj["ContextId"] = Id;
-                obj[nameof(CompileResponse.Diagnostics)] = ConvertToJArray(project.Diagnostics.Diagnostics);
+                obj[nameof(CompileResponse.Diagnostics)] = ConvertToJArray(project.Diagnostics.CompilationDiagnostics);
```

Comment:
Don't know what this is, but how come only the right-hand side changed, and not the left-hand side?